lib/middleman-livereload/extension_3_1.rb

```diff
@@ -13,6 +13,8 @@ class LiveReloadExtension < Extension
     option :js_host, nil, 'Host to connect LiveReload Javascript to (if different than :host)'
     option :livereload_css_target, 'stylesheets/all.css', 'CSS file to load when a @imported CSS partials are modified'
     option :livereload_css_pattern, Regexp.new('_.*\.css'), 'Regexp matching filenames that trigger live reloading target'
+    option :ssl_certificate, nil, 'Path to an X.509 certificate to use for the websocket server'
+    option :ssl_private_key, nil, 'Path to an RSA private key for the websocket certificate'
```

Comment:
I think `wss_certificate` and `wss_private_key` is better. Both because it's not ssl (it's TLS) and we may want to add encryption for the http-part of middleman-livereload in the future.